pkg/charts/linkerd2/values.go

```diff
@@ -29,6 +29,7 @@ type (
 		ControllerImage              string                 `json:"controllerImage"`
 		ControllerReplicas           uint                   `json:"controllerReplicas"`
 		ControllerUID                int64                  `json:"controllerUID"`
+		ControllerGID                *Customisable[int64]   `json:"controllerGID"`
```

Comment:
If we change `gid` to be nullable, we could probably just take a pointer to an int here instead of introducing a new type.